include/crypto/riscv_arch.def

```diff
@@ -32,6 +32,7 @@ RISCV_DEFINE_CAP(ZKSED, 0, 10)
 RISCV_DEFINE_CAP(ZKSH, 0, 11)
 RISCV_DEFINE_CAP(ZKR, 0, 12)
 RISCV_DEFINE_CAP(ZKT, 0, 13)
+RISCV_DEFINE_CAP(V, 0, 14)
```

Comment:
I do not see any code here to set this capability based on a runtime query.

If this is intended, consider clarifying the commit description which can be interpreted to mean that "riscv_vlen()" is valid after this commit is included.
> If this extension is found at run-time, then we read out VLEN as described above and cache it.

Reply:
The code is already in OpenSSL (`crypto/riscvcap.c`).
The patch series is just reusing the existing mechanism.

Reply:
Regarding `riscv_vlen()`: this is addressed by specifying the behaviour of that function if no vector extension is found (i.e. returning 0).